pkg/logstash.sysv

```diff
@@ -96,12 +97,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      if [ $KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
+      if [ $LS_KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
```

Comment:
Actually no.  If you quote a variable and then try to integer compare it with -eq, it generates an error.  However I can should and will change the operator from -eq to == in keeping with the fact that the variable is a string because it wasn't typeset -i.